DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingUsingFunctions/metadata.json

```diff
@@ -0,0 +1,98 @@
+            "Type": "link",
+            "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#projecting-with-object-literals"
+        }
+        /*,
+        TODO nodeJS Docs
+        {
+            "Title": "Query Timeout Configuration",
+            "Type": "link",
+            "Url": "https://ravendb.net/docs/article-page/latest/java/server/configuration/database-configuration#databases.querytimeoutinsec"
+        }
+        */
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define the projecting function",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Within a raw-query-string, define a _JavaScript function_ that processes the 'Employee' document and returns a new 'projected' object.",
+                "",
+                "* The above _output_ function returns a projected object literal that is computed from the following functions:",
+                "  * formatName -  process the 'FirstName' & 'LastName' document fields.<br>",
+                "  * formatTitle - process the document 'Title' field."
+            ],
+            "Assets": [
+                {
+                    "Title": "Projection with declared function - query on collection",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/client-api/session/querying/how-to-project-query-results#example-iv---projection-with-"
+                }
+            ]
+        },
+        {
+            "Title": "Define the RQL query",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Within the raw-query-string, define the `RQL query` that will be executed on the server.",
+                "",
+                "* Use `select` and call the JavaScript function defined from the previous step.",
+                "",
+                "* In the above RQL, a query is defined on the 'Employees' collection.",
+                "  Note the _select_ usage so that returned results are structured according to the defined _output_ method."
+            ],
+            "Assets": [
+            ]
+        },
+        {
+            "Title": "Set the query",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Pass the raw-query-string to the Session `rawQuery()` method."
+            ],
+            "Assets": [
+                {
+                    "Title": "Projection with declared function - query on collection",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/client-api/session/querying/how-to-project-query-results#example-iv---projection-with-"
+                }
+            ]
+        },
+        {
+            "Title": "Execute the query",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* The 'projectedResults' variable only stores the query definition.<br>",
```

Comment:
change:
'projectedResults' => 'projectedQueryWithFunctions'

pls do this for all 3: C#, java & Go

Reply:
pls apply comment